lib/deck.rb

```diff
@@ -0,0 +1,19 @@
+class Deck
```

Comment:
This class is missing a `count` method and required from the specification. I believe this is what `cards_in_deck ` is supposed to be, but the name doesn't match what is in the specification.